assets/js/pages/AdvisoryDetails/AdvisoryDetails.jsx

```diff
@@ -76,10 +76,17 @@ function AdvisoryDetails({
       <div className="flex flex-col mb-4">
         <h2 className="text-xl font-bold mb-2">Fixes</h2>
         <div className="bg-white py-4 px-6 shadow shadow-md rounded-lg">
-          {fixes && fixes.length > 1 ? (
+          {fixes && Object.keys(fixes).length >= 1 ? (
```

Comment:
```suggestion
          {fixes && Object.keys(fixes).length ? (
```

Reply:
Is it common code style here to shorten it like this? Otherwise I'd prefer to keep it like this. I find it much easier to see the intend :D

Reply:
I can see your point but when the motivation is just personal preference we tend to apply fixes rather than sticking with current implementation.

I also tend to lean towards the `?` :smile:

Reply:
Alright ^^